doc/api/fs.md

```diff
@@ -574,6 +574,9 @@ changes:
     description: The `path` parameter can be a WHATWG `URL` object using `file:`
                  protocol. Support is currently still *experimental*.
 -->
+Note that this is a blocking call. Avoid this method in an asynchronous
+environment. See [Overview of Blocking vs Non-Blocking][blocking-vs-non-blocking]
```

Comment:
These lines are 81 characters now, so I am not sure the doc linting would be happy with it. Maybe the link can be just `[Overview of Blocking vs Non-Blocking][]` and the bottom reference `[Overview of Blocking vs Non-Blocking]: https://...`?

Reply:
Here are some details: https://daringfireball.net/projects/markdown/syntax#link